bin/setup_exercism_local_aws

```diff
@@ -19,7 +19,7 @@ unless ENV["EXERCISM_SKIP_S3"]
     Exercism.config.aws_tooling_jobs_bucket,
   ].each do |bucket|
     begin
-      ExercismConfig::SetupS3Client.().create_bucket(bucket: bucket)
+      Exercism.config.dynamodb_client.create_bucket(bucket: bucket)
```

Comment:
This shouldn't have changed.

Reply:
(ie, you've changed it from s3 to dynamodb)

Reply:
Actually, at [line 10](https://github.com/exercism/config/blob/4c7b590d247807a876efd50cdd4e7d6cb89c3498/bin/setup_exercism_local_aws#L10) this is done:

```ruby
Exercism.config.dynamodb_client = ExercismConfig::SetupDynamoDBClient.()
```

So I think the previous iteration was not using that.